pkgs/development/python-modules/gpib-ctypes/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytestCheckHook
+, pytest-runner
+}:
+
+buildPythonPackage rec {
+  version = "0.3.0";
+  pname = "gpib_ctypes";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-c9l6TNmM4PtbvopnnFi5R1dQ9o3MI39BHHHPSGqfjCY=";
+  };
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    pytest-runner
+  ];
+
+  pythonImportsCheck = [
+    "gpib_ctypes.gpib"
+  ];
+
+  meta = with lib; {
+    description = "Cross-platform Python bindings for the NI GPIB and linux-gpib C interfaces";
+    longDescription = ''
+      To utilize the installed GPIB library and access GPIB devices, the required shared library
+      files must be located in the following directories:
+
+      For Windows: c:\\windows\\system32\\gpib-32.dll
+
+      For Linux: /usr/lib/libgpib.so.0 or
+      /usr/lib/local/libgpib.so.0
```

Comment:
can you not package that for linux aswell and use patchelf on the library to fix the paths?

Reply:
Yes, package is pending at PR #245677 . Requesting for your help to merge.

Reply:
@Janik-Haag I have sent you an email regarding the `linux-gpib` package, requesting for your help to review and merge

Reply:
I reviewed and merged the other PR. so we can omit this now. I'm also going to answer your email.